### ISSUE TITLE
Source code missing

### ISSUE THREAD

**Issue by roberto-mainetti** [EmBitz/EBlink#25]
It's a very nice project, but on Github I think you should publish the source code, not the executable

**Comment by EmBitz** [comment]
Yes, I feel what you mean and I also struggled with it. However, I saw other projects doing the same but perhaps I should change this,

This project doesn't focus on the binary code but on scripts. It doesn't try to be another OpenOCD, pyOCD or Texane. The purpose for github is/was sharing the scripts and modify them. The scripts are the assets. However, to give users an easy way to use it, it is convenient that the executables are in place. 

Perhaps this should be removed from github if it doesn't belong here.

Thanks!
